src/pages/components/avatars/PreviewSvelte.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import Avatar from "@components/Avatar/Avatar.svelte";
+
+</script>
+
+<div class="flex space-x-4">
+
+    <Avatar imageSrc="https://images.unsplash.com/photo-1617296538902-887900d9b592?ixid=M3w0Njc5ODF8MHwxfGFsbHx8fHx8fHx8fDE2ODc5NzExMDB8&ixlib=rb-4.0.3&w=128&h=128&auto=format&fit=crop" 
```

Comment:
Super nitpick here - but `w-32` is a bit large for avatars. Let's go down to something like `w-16`, which should be the default.